pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -64,8 +64,7 @@
   ENTRIES_SCANNED_IN_FILTER("documents", false),
   ENTRIES_SCANNED_POST_FILTER("documents", false),
 
-  NUM_RESIZES("resizes", false),
-  RESIZE_TIME_MS("resizeTimeMs", false),
+  NUM_RESIZES("numResizes", false),
```

Comment:
@siddharthteotia I also change the unit of BrokerMeter from "resizes" to "numResizes" to keep consistence with ServerMeter. Let me know if I need change back.